aptos-move/framework/aptos-framework/sources/storage_gas.spec.move

```diff
@@ -77,4 +124,23 @@ spec aptos_framework::storage_gas {
     // ---------------------------------
 
     spec fun spec_calculate_gas(max_usage: u64, current_usage: u64, curve: GasCurve): u64;
+
+    spec schema AbortsIfNewGasCurve {
+        min_gas: u64;
+        max_gas: u64;
+
+        aborts_if max_gas < min_gas;
+        aborts_if max_gas > MAX_U64 / BASIS_POINT_DENOMINATION;
+    }
+
+    /// A non decreasing curve must ensure that next is greater than cur.
+    spec schema AbortsIfValidatePoints {
```

Comment:
The convention is `ValidatePointsAbortsIf`.

Reply:
The modification is complete.

Reply:
Done.